languages/swift/exercises/concept/tuples/.docs/after.md

```diff
@@ -0,0 +1,84 @@
+[Tuples][tuples] are used to combine multiple values into a single compound value where each of the values may have a different type. The compound value has a type that is the combination of the individual values' types.
+
+```swift
+let order: (String, Double, Int) = ("biscuits", 2.99, 3)
+```
+
+The fact that tuples are single values allows them to be used to [return multiple values from a function][multiple-return-values]:
+
+```swift
+func longestAndShortest(_ strings: [String]) -> (longest: String?, shortest: String?) {
+  var longestLength = Int.min, shortestLength = Int.max
+  var longAndShort: (longest: String?, shortest: String?) = (longest: nil, shortest: nil)
+  for string in strings {
+    if string.count > longestLength {
+      longAndShort.longest = string
+      longestLength = string.count
+    }
+    if string.count < shortestLength {
+      longAndShort.shortest = string
+      shortestLength = string.count
+    }
+  }
+  return longAndShort
+}
+
+longestAndShortest(["robin", "blue jay", "heron", "swift", "swan"])
+// => (longest: "blue jay", shortest: "swan")
+```
+
+The individual items of a tuple may be accessed by appending a `.n` to the tuple's name where _n_ is the index of the element you would like to access, starting with 0.
```

Comment:
As here starts a "recap" (what has been learned from introduction and the exercise) what about inserting a `Key Takeaways from this Exercise` heading here like in https://github.com/exercism/v3/blob/master/languages/swift/exercises/concept/dictionaries/.docs/after.md#key-takeaways-from-this-exercise?

Reply:
I think that exercise is the only one that uses that format, the rest uses this format.